### ISSUE TITLE
SteamVR Crashes when starting PSMoveService

### ISSUE THREAD

**Issue by ThePeculiarHalo** [HipsterSloth/PSMoveSteamVRBridge#93]

**Comment by manobroda** [comment]
I'm having a similar issue. Suddenly I'm getting a 108 error from SteamVR and the message "Failed to receive UDP connection id: No connection could be made because the target machine actively refused it" from Psmoveservice.
One day was working perfectly, other it apears these errors.
I already spend 2 days trying to solve this issue, but I got no solutions.
I suspect that the problem came with a windows 10 update. Another Bill Gate's headeache!

**Comment by ThePeculiarHalo** [comment]
So here's the funny part, I was looking at buying driver4vr, so I downloaded the trial and started messing with it, and the psmoveservice worked through that. I later uninstalled it and for some reason the service still works, I haven't had it crash in a month, and I didn't have to buy driver4vr. Maybe it'll work for you? I'm not even sure how it happened, but it did

**Comment by manobroda** [comment]
I just figured out my problem.
It was a joystick (wheel and pedals) connect in USB that it was been detect as a psnavi. Can you beleave this?

Anyway, after disconnect this joystick, psmove service don't send more error messages.

I recomend unplug all other devices USB.
If you found the solution, please teel us.

I founded that camera's drivers are crashing psmove service too. So I recomend reinstall or change this drivers.

Regards.

**Comment by AgoniXX** [comment]
> I just figured out my problem.
> It was a joystick (wheel and pedals) connect in USB that it was been detect as a psnavi. Can you beleave this?
You are right! My Xbox360 joystick caused this issue too. Disconnected it and no more crashes!! Thanks!!
